pkg/api/customization/cluster/actionhandler.go

```diff
@@ -75,8 +75,14 @@ func (a ActionHandler) ClusterActionHandler(actionName string, action *types.Act
 		}
 		return a.disableMonitoring(actionName, action, apiContext)
 	case "backupEtcd":
+		if !canUpdateCluster() {
+			return httperror.NewAPIError(httperror.Unauthorized, "can not backup etcd")
```

Comment:
Can we add the ID of this request's caller to the error message?

Reply:
I am not sure if that's necessary, the logged in user will already be getting this error message